app/views/lists/edit.html.slim

```diff
@@ -0,0 +1,6 @@
+= provide(:button_text, 'Edit List')
+= provide(:cancel_btn_text, 'Cancel')
+= provide(:path, list_path)
+h1.title
+  | Edit List profile
+= render 'form', list: @list
```

Comment:
missing the final blank line at the end of the file

Reply:
done